tests/conftest.py

```diff
@@ -0,0 +1,83 @@
+import datetime
+import random
+import typing
+
+import pytest
+
+from src.github import Episode, Repo, parse_issue_markdown
+
+
+@pytest.fixture(scope="session")
+def date():
+    return datetime.datetime.today()
+
+
+@pytest.fixture(scope="session")
+def time():
+    return datetime.time(
+        hour=random.randint(0, 23),
+        minute=random.randint(0, 59),
+        second=random.randint(0, 59),
+    )
```

Comment:
In general, I'd avoid random data like this in tests, which can lead to flaky tests that result in hard-to-reproduce failures. Tools like [hypothesis](https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.datetimes) can allow for robust testing where multiple values are required.

Reply:
honestly I just needed a time... Haven't played with hypothesis. can definitely take a look